server/models/pattern/patterns/patterns.go

```diff
@@ -45,34 +48,68 @@ func ProcessOAM(kconfigs []string, oamComps []string, oamConfig string, isDel bo
 		wg.Add(1)
 		go func(kcli *kubernetes.Client) {
 			defer wg.Done()
+			id, _ := uuid.NewV4()
 			for _, comp := range comps {
+				var req meshes.EventsResponse
 				if comp.Spec.Type == "Application" {
 					if err := application.Deploy(kcli, comp, config, isDel); err != nil {
+						req = meshes.EventsResponse{
+							EventType:   meshes.EventType_ERROR,
+							Summary:     "Error deploying application: " + comp.Name,
+							Details:     err.Error(),
+							OperationId: id.String(),
+						}
 						errs = append(errs, err)
+						continue
 					}
-
 					if !isDel {
+						req = meshes.EventsResponse{
+							EventType:   meshes.EventType_INFO,
+							Summary:     "Successfully deployed application " + comp.Name,
```

Comment:
@Revolyssup Can we be consistent wrt message format `Successfully deployed application: " + comp.Name,`, as in UI it will be easier to get the design name, or component name